src/api/app/views/webui/request/_actions_details.html.haml

```diff
@@ -3,8 +3,8 @@
   submit_actions_count = submit_actions.count
 
 - if submit_actions_count > 1
-  .d-flex.align-items-center
-    %p.mb-0 This request contains #{submit_actions_count} actions
+  .d-flex.flex-wrap.mb-3.align-items-center
```

Comment:
@danidoni, I forgot to add this class `mb-3`. Please take a look again.

Reply:
LGTM